zkevm-circuits/src/evm_circuit/execution/error_oog_log.rs

```diff
@@ -255,49 +290,48 @@ mod test {
             .write_op(terminator)
         };
 
-        Account::mock_100_ether(bytecode)
+        Account {
+            address: Address::repeat_byte(0xfe),
+            balance: Word::from(10).pow(20.into()),
+            code: bytecode.code().into(),
+            ..Default::default()
+        }
     }
 
-    fn oog_log_internal_call(caller: Account, callee: Account) {
-        let ctx = TestContext::<3, 1>::new(
-            None,
-            |accs| {
-                accs[0]
-                    .address(address!("0x000000000000000000000000000000000000cafe"))
-                    .balance(Word::from(10u64.pow(19)));
-                accs[1].account(&caller);
-                accs[2].account(&callee);
-            },
-            |mut txs, accs| {
-                txs[0]
-                    .from(accs[0].address)
-                    .to(accs[1].address)
-                    .gas(24000.into());
-            },
-            |block, _tx| block.number(0xcafeu64),
-        )
-        .unwrap();
-
-        CircuitTestBuilder::new_from_test_ctx(ctx).run();
+    fn callee(code: Bytecode) -> Account {
+        let code = code.code();
+        let is_empty = code.is_empty();
+        Account {
+            address: Address::repeat_byte(0xff),
+            code: code.into(),
+            nonce: if is_empty { 0 } else { 1 }.into(),
+            balance: if is_empty { 0 } else { 0xdeadbeefu64 }.into(),
+            ..Default::default()
+        }
```

Comment:
```suggestion
        Account::mock_code_balance(code)
```

Reply:
updated in [2236916](https://github.com/privacy-scaling-explorations/zkevm-circuits/pull/1564/commits/22369163147c2f20c622f8521a59b1ef24e13352) thanks !